src/Kernel32.Desktop/Kernel32Extensions.cs

```diff
@@ -14,16 +14,17 @@ public static partial class Kernel32Extensions
         /// Gets the text associated with an <see cref="NTSTATUS"/>.
         /// </summary>
         /// <param name="status">The error code.</param>
+        /// <param name="dwLanguageId">The language indentifier for the message</param>
         /// <returns>The error message. Or <c>null</c> if no message could be found.</returns>
-        public static string GetMessage(this NTSTATUS status)
+        public static string GetMessage(this NTSTATUS status, int dwLanguageId = 0)
```

Comment:
Really? I had understood form before that it was better not to create an overload and use new parameter with default values in this case... All the times I had created overloads we changed to optional parameters.